src/services/configurations/index.js

```diff
@@ -4,11 +4,12 @@ import { checkStatus, parseJSON } from "../common";
 
 const config = {
     path: process.env.PUBLIC_URL + "/configurations/",
+    api: process.env.REACT_APP_API_URL,
```

Comment:
I think its fine to have process.env.REACT_APP_API_URL, as we build the app for production, then this will going to be replaced.